test_scenarios/UC8_Show owned CN list.md

```diff
@@ -2,7 +2,38 @@
 
 ## UC8 Show owned CN list
 
-### Główny 
+### Sprawdzane UC
 
-##### 1. __App User__ wybiera opcję wyświetlenia listy posiadanych CN'ów.
-##### 2. System wyświetla listę posiadanych CN'ów użytkownika (ich nazwę, parametry i status obliczania)
\ No newline at end of file
+- [UC8](../scenarios/UC8_Show%20owned%20CN%20list)
+
+### Warunki początkowe 
+
+- Zalogowany __Supplier__ to _user01_.
+
+### Przypadki testowe
+
+#### 1. Pomyślne wyświetlenie listy posiadanych maszyn.
+
+1. __Supplier__ wybiera opcję wyświetlenia posiadanych CN
+2. System wyświetla listę posiadanych CN - pokazując ich nazwy oraz statusy
```

Comment:
Scenariusze do tego UC nie obejmowały pobrania statusu. W takim razie jezeli ma byc odpytywanie o status jak ma wyglądać odpytanie? W swaggerze mamy zdefiniowane to jako GET na interfejsie /machine/status wystawianym przez Machine. Jak rozróżniamy tutaj maszyny, nie powinno być jakiegoś id?

Reply:
Masz rację, w ramach tego testu statusy mogą być zamockowane.
Docelowo Machine Manager powinien mieć aktualną informację o stanie wszystkich zarejestrowanych maszyn. Sprawdzanie może odbywać się przez cykliczne sprawdzanie `machine.status` - jeśli chodzi o implementację (skąd wziąc ip maszyny) decyzja należy do dewelopera